README.md

```diff
@@ -16,3 +18,45 @@ In particular the objectives of the device are:
  - quick interaction with an easy human-machine interface
  - permanent feedback on which task is active at the moment
  - provide digital interface to recorded times (TBD: through WiFi, Bluetooth, wired serial interface, ...)
+
+## Generate the documentation
+
+Documentation may be generated with Doxygen. Simply call in the root directory of this repository:
+
+    doxygen
+
+The HTML documentation will be written to [`doc/html/index.xhtml`](doc/html/index.xhtml).
+
+## Build
+
+The build configuration for the device's software is generated with PlatformIO. Please refer to the [documentation](https://docs.platformio.org/) for build instructions.
+
+## Simulate
+
+[`wokwi.toml`](wokwi_files/wokwi.toml) is a project configuration for the Visual Studio Code extension [Wokwi Simulator](https://marketplace.visualstudio.com/items?itemName=wokwi.wokwi-vscode).
+Please refer to its documentation for instructions on how to simulate the device.
+
+## Contribute
```

Comment:
Should be moved to a separate CONTRIBUTION.md file

Reply:
Done